submissions/7SergSmith7/DOM model/src/html/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>DOM model Task</title>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h1>Top 5 search engines</h1>
+      </header>
+
+      <main>
+        <nav class="navigation">
+          <h2 class="navigation__header">Services</h2>
+          <ul class="navigation__menu">
+            <li class="menu__item">
```

Comment:
It's ok for this task. But you can generate this navigation menu in your script file

Reply:
Thank you for advice!
I need to fix it to generate from js?

Reply:
It's up to you. As your app is pretty much simple you can add it. It will become more data-driven.

Reply:
I am afraid that I will not have time to complete all the tasks before the deadline.
I will try to improve my application after completing the rest of the tasks.
What else do I need to do to get my PR approved?

Reply:
Ok, no problem.
Now I see a lot of commits from the previous tasks in this PR.
Do we need to keep them here?